packages/browser-lite/src/mesh.ts

```diff
@@ -129,6 +129,9 @@ window.addEventListener(loadEventName, () => {
     isWasmLoaded = true;
 });
 
+const schemaValidator: any = {};
```

Comment:
This creates a race condition. Here, `window.schemaValidator` is defined before `schemaValidator.orderValidator` and `schemaValidator.messageValidator` are set. In __validate_js.go__, we only check that `window.schemaValidator` is not `null` or `undefined` or `null` and then we immediately call `orderValidator` and `messageValidator`, without checking if _those_ are `undefined` or `null`.

It also creates additional complexity since there are two ways of setting `window.schemaValidator`, one here and one in __browser_shim.ts__. That can potentially cause bugs down the road if we change one but forget to change the other.

Here's what I think we should do instead:

1. Move `setSchemaValidator` from __browser_shim.ts__ into __schema_validator.ts__. __browser_shim.ts__ should just import the existing function and assign it to `window`.
2. Use the same function (i.e. `setSchemaValidator`) whether we are loading the schema validator in `@0x/mesh-browser`, `@0x/mesh-browser-lite`, or for our tests.
3. Eliminate the race condition by not setting `window.schemaValidator` before setting `orderValidator` and `messageValidator`.

Step 3 might require the addition of a new global variable to indicate to the Go code that the schema validator will be loaded (e.g. similar to what we did for BrowserFS). Or better yet, we could restructure things so that the schema validator is passed through to Go code via `core.New`.

Reply:
Also the type of `schemaValidator` should be `SchemaValidator`, not `any`.

```suggestion
const schemaValidator: SchemaValidator = {};
```

Or depending on what you do, it might need to be `SchemaValidator | null`.